.github/workflows/i18n-unused-keys.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
           # Define paths
           I18N_FILE="client/src/locales/en/translation.json"
-          SOURCE_DIRS=("client/src" "api")
+          SOURCE_DIRS=("client/src" "api" "packages/data-provider/src")
```

Comment:
Why is this needed? data-provider doesn't have any i18n keys does it?

Reply:
Because [parameterSettings](https://github.com/danny-avila/LibreChat/pull/7342/files/a36f58467dd217fa832e81920d55d07d5923e6fd#diff-f6a5533d3123073da033a3462eee314c8af6118b55d78e22f26039dc95e70514) has been moved from frontend to data-provider so that the backend can use it to generate overriden params and validate their definitions.

This really should be the responsibility of the backend, and the frontend should only be responsible for displaying a list of params given to it (i.e. the param selection/parsing logic should be handled on the backend). It's related to this [comment](https://github.com/danny-avila/LibreChat/pull/7342#discussion_r2089279647) by @sbruel. I'm creating a follow-up issue to handle this isolation-of-concerns issue.